test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs

```diff
@@ -690,6 +690,117 @@ public void InstallCommand_FromPackagesConfigFile_SpecifyingRelativeSolutionDir(
             }
         }
 
+        [Fact]
+        public void InstallCommand_FromPackagesConfigFile_PackageSaveModeNuspec()
```

Comment:
You could have a theory with the package save mode here. 

The tests are largely equivalent.

Reply:
Merge `InstallCommand_FromPackagesConfigFile_PackageSaveMode*()` into one? But the assertions are different.

Reply:
I might misunderstand. You mean merge `InstallCommand_FromPackagesConfigFile_PackageSaveModeNuspec()` and `InstallCommand_PackageSaveModeNuspec()` into one, right?

Reply:
You can always have conditional assertions. 

For example: https://github.com/NuGet/NuGet.Client/blob/b58613fbc338b8e142d29b3bdb74e92a83a259fb/test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs#L1029-L1053

Reply:
thanks for the hint.